www/js/controller.js

```diff
@@ -640,15 +583,18 @@ define([
 			 Set value for  value, equation (input),
 			 */
 
-
 			if(model.getNodeIDFor){
 				var d = registry.byId(this.controlMap.description);
 				array.forEach(this._model.authored.getDescriptions(), function(desc){
 					var exists =  model.getNodeIDFor(desc.value);
-					d.getOptions(desc).disabled=exists;
-					if(desc.value == nodeName){
-						d.getOptions(desc).disabled=false;
-					}});
+					
+					if(d.getOptions(desc)) {
+						d.getOptions(desc).disabled=exists;
+						if(desc.value == nodeName){
+							d.getOptions(desc).disabled=false;
+						}
```

Comment:
is there a scenario reason here for it to be setting disable for a particular field explicitly? This should come from PM and applyStatusDirective should do it right?